contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -560,7 +562,7 @@ format_errmsg(const char *fmt, const char *arg1, const char *arg2);
 
 inline std::u32string utf8_to_utf32(const char* s)
 {
-	return antlrcpp::utf8_to_utf32(s, s + strlen(s));
```

Comment:
What's the difference between those two functions?

Reply:
utf8_to_utf32 was removed in v4.10 and replaced with lenient/strict decode https://github.com/antlr/antlr4/commit/2349fb20199303eef2a742f9449d73c4902e83c9

I opted to go with lenientDecode in order to prevent errors from being raised, but we can also change this to strictDecode if that behavior is more appropriate